### ISSUE TITLE
How to add 2nd gotify Server?

### ISSUE THREAD

**Issue by nicedevil007** [androidseb25/iGotify-Notification-Assistent#113]
Hey Seb, 

I just wanted to ask on how this should be done? If I remeber Right you Asses the ability to add multiple gotify instances to the iOS App. How can this be achieved?

I have 2 instances now: one from work and one from private homelab. Both are Running gotify and iGotify but I only have one iPhone. I want to see pushnotifications from both setups.

Can you create Wiki entry to clear this up please ❤️

**Comment by androidseb25** [comment]
Hi,

you can add a second instance via the notification tab and on the top you will see a little arrow an there will be push a submenu with the add point take a look at the screenshot.


![image](https://github.com/user-attachments/assets/3ccbeae5-114c-4ef0-8836-b6190163bf0a)

**Comment by nicedevil007** [comment]
Hey Seb, thank you for this hint :)

I was always tring to find this on this part of your app:

![image](https://github.com/user-attachments/assets/1437b48b-2573-4706-9c92-4f13cbff3033)

![image](https://github.com/user-attachments/assets/92d5e895-8135-41f6-aa90-215b529e9f3f)

So I would like to raise a feature request to add the option here as well "Add an instance" :)

**Comment by androidseb25** [comment]
Yeah this option under settings comes with the next release ;) I already added it 😅

And no "local instance" mean that the igotify container is not available for the outside network, e.g. he is connected only via IP in the local network. That the reason why you getting in the setup process the configuration information for the container with the environment variables.
